src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1370,4 +1370,7 @@
   <data name="EnumShouldNotHaveDuplicatedValuesTitle" xml:space="preserve">
     <value>Enums values should not be duplicated</value>
   </data>
+  <data name="DoNotIgnoreMethodResultsMessageDisposable" xml:space="preserve">
+    <value>{0} calls {1} which is never disposed. Ensure this is the intended use.</value>
```

Comment:
`MyMethod calls GetSomethingDisposable which is never disposed. Ensure this is the intended use.`

maybe it would be better with

`'MyMethod' calls 'GetSomethingDisposable' whose value is never disposed. Ensure this is the intended use.`